guidelines/project-best-practices.md

```diff
@@ -0,0 +1,197 @@
+  - https://github.com/hyperledger/cacti/blob/main/CONTRIBUTING.md
+  - https://hyperledger-fabric.readthedocs.io/en/latest/CONTRIBUTING.html
+  - https://wiki.hyperledger.org/display/indy/How+to+Contribute
+  - https://hyperledger.github.io/firefly/contributors/
+  - https://github.com/hyperledger/iroha/blob/main/CONTRIBUTING.rst
+* Present at Meetups - Virtual or in person – these are well attended and the videos also get many views
+  - Email meetup@hyperledger.org if you're interested in presenting or join one of the [bi-weekly Meetup and Workshop planning calls every other Thursday at 9:30 AM pacific](https://lists.hyperledger.org/g/meetups/calendar)
+* Host technical Workshops - Virtual or in person (e.g. Global Forum) – these are well attended and the videos also get many views
+  - Reach out to one of the Hyperledger Community Architects or join one of the [bi-weekly Meetup and Workshop planning calls every other Thursday at 9:30 AM pacific](https://lists.hyperledger.org/g/meetups/calendar)
+* Take part in annual Mentorship program
+  - Near the beginning of each year maintainers have the option to [submit projects to the annual Hyperledger mentorship program](https://wiki.hyperledger.org/display/INTERN/Mentorship+Projects) and work with mentees or code, documentation or research projects
+* See also [Raising the Profile of your Hyperledger Project or Lab](https://docs.google.com/presentation/d/13nji_R-op77ERT-AV3-CbOOZwAOjtvq33RRGnjpL3Gc/edit#slide=id.g3cf4a9ebda_0_647)
+
+## Project management and issues
+
+* Maintain a written project roadmap, discuss in project meetings
+* Create, clarify, and label issues in Github for contributors. Use Github default labels, e.g. "good first issue"
+  - Consider splitting "good first issue" label into multiple labels, e.g. "good-first-issue-100-introductory" through "good-first-issue-400-expert" (see Cacti project)
+* Review, triage, comment on, and close inbound Github issues
+
+## Pull Request management
+
+* Quick review turnarounds are appreciated and encourage future contributions (and shows up in Insight reports).
+* Equal attention to PRs - review in order of arrival as a general rule of thumb.
+* 'Over'-communicate in PR comments, especially if review is delayed - contributors don't know what is in a maintainer's head
+* Be gentle on new contributors, perhaps relax coding guidelines and fix up later
+* Don't leave contributors hanging... if the contribution is not a good fit say so
+* Mentor new contributors through the process, in PRs and otherwise
+
+## Security
+
+* Provide three named security contacts per project
+* Define security issue reporting process in SECURITY.md with reference to [Hyperledger reporting process](https://wiki.hyperledger.org/display/SEC/Defect+Response)
+* Review, respond, and act on reported security vulnerabilities
+* Follow security issue disclosure process - see [Disclosure task force](https://github.com/hyperledger/toc/issues/48)
```

Comment:
[nitpick] To be updated after the outcome of the task force.

Reply:
Absolutely, I view this as a 'living document' that we'll update as task forces complete or we otherwise learn new information.